Timeline/src/index.ts

```diff
@@ -174,27 +194,28 @@ export default class CjaasTimelineWidget extends LitElement {
         headers: {
           "content-type": "application/json; charset=UTF-8",
           accept: "application/json",
-          Authorization: `SharedAccessSignature ${this.streamToken}`
-        }
+          Authorization: `SharedAccessSignature ${this.streamToken}`,
+        },
       };
       this.eventSource = new EventSource(
         `${this.baseURL}/v1/journey/streams/${this.personId}?${this.streamToken}`,
-        header
+        header,
       );
     }
 
     if (this.eventSource) {
-      this.eventSource!.onmessage = (event: ServerSentEvent) => {
+      this.eventSource.onmessage = (event: ServerSentEvent) => {
         let data;
         try {
           data = JSON.parse(event.data);
+          data.time = DateTime.fromISO(data.time);
           this.newestEvents = [data, ...this.newestEvents];
         } catch (err) {
           console.log("Event Source Ping");
         }
       };
 
-      this.eventSource!.onerror = () => {
+      this.eventSource.onerror = () => {
```

Comment:
why remove the exclamation mark, doesn't that help prevent calling on an undefined object?

Reply:
we check for this in line 186 `    if (this.eventSource) {` already

Reply:
ok